tuf/client/updater.py

```diff
@@ -2883,9 +2883,10 @@ def _ensure_not_expired(self, metadata_object, metadata_rolename):
     # If the current time has surpassed the expiration date, raise
     # an exception.  'expires' is in 'tuf.formats.ISO8601_DATETIME_SCHEMA'
     # format (e.g., '1985-10-21T01:22:00Z'.)  Convert it to a unix timestamp and
-    # compare it against the current time.time() (also in Unix/POSIX time
-    # format, although with microseconds attached.)
-    current_time = int(time.time())
+    # compare it against the current time from tuf.util.get_current_time(),
+    # which is also in Unix/POSIX time format, although with microseconds
+    # attached, and allows manual override of the system clock.)
```

Comment:
Aren't you cutting of microseconds with the `int` cast?

Reply:
Ah, right, that's no longer being done in this section, but now in the called function, so I should remove that note.